Eatery/Controllers/Eateries/GivingDayViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  GivingDayViewController.swift
+//  Eatery
+//
+//  Created by Gonzalo Gonzalez on 3/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class GivingDayViewController: UIViewController {
+
+    var givingDayPopupView: GivingDayView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        //blur effect setup
+        if !UIAccessibility.isReduceTransparencyEnabled {
+            view.backgroundColor = .clear
+            view.isOpaque = false
+            let blurEffect = UIBlurEffect(style: .dark)
+            let blurEffectView = UIVisualEffectView(effect: blurEffect)
+            blurEffectView.frame = self.view.bounds
+            blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+            
```

Comment:
delete space

Reply:
Constraints for the blur view?

Reply:
> Constraints for the blur view?

The constraints for the blur view are the bounds of the view (line 24). I was unable to make constraints using Snapkit, so that will have to do.

Reply:
Attempting to use the constraints results in a
Thread 1: Fatal error: Unexpectedly found nil while unwrapping an Optional value

Reply:
It was stuck inside the scope of the if statement. Setting it as a global variable allowed me to properly make constraints.